test/serve/serve-variable/webpack.config.js

```diff
@@ -0,0 +1,24 @@
+const isInProcess = process.env.WEBPACK_SERVE;
```

Comment:
We don't need to assign the property on the process object?

Reply:
Yes

Reply:
We should document then that previously the var was on process object now it lives inside the env supplied to config

Reply:
Yes, I already have mentioned in PR that this is a breaking change. Will add the bit about enforcement of use of `env` argument as well.

Reply:
Yes, let's update it after merge, I will open an issue